### PR TITLE
DNM - BREAK CI introducing an issue in the glance-operator

### PR DESCRIPTION
**Pull request by fmount**
Breaking CI

### CODE REVIEW

File: controllers/glanceapi_controller.go [hunk 1]
```diff
@@ -573,13 +573,13 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
...
 	}
-	keystonePublicURL, err := keystoneAPI.GetEndpoint(endpoint.EndpointPublic)
+	//keystonePublicURL, err := keystoneAPI.GetEndpoint(endpoint.EndpointPublic)
 	if err != nil {
 		return err
 	}
 	templateParameters := make(map[string]interface{})
 	templateParameters["ServiceUser"] = instance.Spec.ServiceUser
-	templateParameters["KeystoneAuthURL"] = keystonePublicURL
+	templateParameters["KeystoneAuthURL"] = "keystonePublicURL"
```
Comment (fmount) [inline controllers/glanceapi_controller.go:582]:
```
2023-04-03T15:47:31+02:00       ERROR   Reconciler error        {"controller": "glanceapi", "controllerGroup": "glance.openstack.org", "controllerKind": "GlanceAPI", "GlanceAPI": {"name":"glance-internal","namespace":"openstack"}, "names
pace": "openstack", "name": "glance-internal", "reconcileID": "850d9df9-1a67-4c2c-a361-0d229b8b7797", "error": "Deployment.apps \"glance-internal-api\" is invalid: [spec.template.spec.initContainers[0].env[0].valueFrom.secretKeyRef.key:
Required value, spec.template.spec.initContainers[0].env[1].valueFrom.secretKeyRef.key: Required value]"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
        /home/fmount/golang-book/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:329
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
        /home/fmount/golang-book/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:274
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
        /home/fmount/golang-book/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:235
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciling Service 'glance-external'
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciling Service 'glance-external' init
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Service glance-public - updated
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Route glance-public - updated
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciled Service 'glance-external' init successfully
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciling Service 'glance-external' update
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciled Service 'glance-external' update successfully
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciling Service 'glance-external' upgrade
2023-04-03T15:47:31+02:00       INFO    controllers.GlanceAPI   Reconciled Service 'glance-external' upgrade successfully
```

which lead to:

```
> oc get glance
NAME     STATUS    MESSAGE
glance   Unknown   Setup started
```

@Sandeepyadav93 is that ok that we don't reach a positive status or we want to see the pod going in crashloopback?

**Comment by Sandeepyadav93** [comment]
@fmount hi o/

This seems to be failing as expected, thanks for testing this.

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_glance-operator/194/pull-ci-openstack-k8s-operators-glance-operator-master-glance-operator-build-deploy/1643183323884818432/artifacts/glance-operator-build-deploy/openstack-k8s-operators-gather/artifacts/get_all.log
~~~
pod/glance-operator-controller-manager-5d85b6f668-q58lz               1/2     CrashLoopBackOff   9 (2m28s ago)   23m
~~~

**Comment by fmount** [comment]
Closing as it works as expected.

**Pull request closed without merging**